concrete/src/Package/Package.php

```diff
@@ -770,38 +1086,35 @@ public function upgradeDatabase()
      */
     public function getNamespace($withLeadingBacksalsh = false)
     {
-        $leadingBkslsh = '';
-        if ($withLeadingBacksalsh) {
-            $leadingBkslsh = '\\';
-        }
+        $leadingBkslsh = $withLeadingBacksalsh ? '\\' : '';
```

Comment:
`withLeadingBackslash`

Reply:
It was already `$withLeadingBacksalsh` ([link](https://github.com/concrete5/concrete5/blob/2928b2f14cb0be34c1de8fb19640a2b4d8ead9e4/concrete/src/Package/Package.php#L771)), but it's better to fix this. Thanks!